exercises/diffie-hellman/src/main/kotlin/DiffieHellman.kt

```diff
@@ -0,0 +1,3 @@
+class DiffieHellman {
```

Comment:
Do we actually need class here? Seems like `object` will be enough for now.

Reply:
We already have almost a half of exercises implemented by  `object`. Let's start [reducing this amount](https://github.com/exercism/kotlin/pull/339#discussion_r345259383) (, please!). I'd rather recomment to treat a `class`-based implementation as a default one.

Reply:
I don't see a need for state-less classes here, generally. It's not an idiomatic Kotlin. :) Top-level function is better but some use `object`s as namespaces (since Kotlin don't have namespaces).